Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM node:16.3-slim
-RUN apt-get update && apt-get install git g++ make python -y
+RUN apt-get update && apt-get install git g++ make python tree -y
```

Comment:
What's `tree` needed for?

Reply:
I was testing this and wanted to be able to dump the directory structure to the logs to ensure the build got put in the right place. Not needed anymore, so I can remove, but it also doesn't hurt anything.